core/src/test/java/apoc/index/SchemaIndexTest.java

```diff
@@ -269,39 +435,35 @@ public void testDistinctWithMultiLabelFullTextIndexShouldNotHangs() {
         });
 
         testResult(db, SCHEMA_DISTINCT_COUNT_ORDERED, map("label", "", "key", ""), (result) -> {
-            extractedFoo(result);
-            extractedFullTextFullTextOneProp1(result);
-            extractedFullTextFullTextOneProp3(result);
-            assertDistinctCountProperties(FULL_TEXT_TWO_LABEL, "prop1", List.of("Ryan"), 2L, result);
-            assertDistinctCountProperties(FULL_TEXT_TWO_LABEL, "prop1", List.of("omega"), 1L, result);
-            assertDistinctCountProperties(FULL_TEXT_TWO_LABEL, "prop3", List.of("abcde"), 1L, result);
-            extractedPerson(result);
+            extractEverything(result);
             assertFalse(result.hasNext());
         });
 
         testResult(db, SCHEMA_DISTINCT_COUNT_ORDERED, map("label", FULL_TEXT_LABEL, "key", ""), (result) -> {
             extractedFullTextFullTextOneProp1(result);
+            extractedFullTextFullTextOneProp2(result);
             extractedFullTextFullTextOneProp3(result);
             assertFalse(result.hasNext());
         });
 
         db.executeTransactionally("DROP INDEX fulltextComposite");
     }
 
+    // This test is run on a separate db as the node it creates sometimes could be seen by other tests run in parallel
```

Comment:
This can probably be fixed with a migration to Junit5 and using:

@ImpermanentEnterpriseDbmsExtension(createDatabasePerTest = true)

But that can be handled later, just so you know :)